packages/components/src/interfaces/internal/Navbar.ts

```diff
@@ -51,39 +54,79 @@ const NavbarItemSchema = Type.Object({
   ),
 })
 
-export const NavbarSchema = Type.Object({
-  items: Type.Array(NavbarItemSchema, {
-    title: "Navbar items",
-    description: "List of items to be displayed in the navbar",
-  }),
-  callToAction: Type.Optional(
-    Type.Object({
-      label: Type.String({
-        title: "Label for the call to action button",
-        maxLength: 30,
-      }),
-      url: Type.String({
-        title: "URL destination of the call to action button",
-        format: "link",
-      }),
+export const NavbarSchema = Type.Object(
+  {
+    items: Type.Array(NavbarItemSchema, {
+      title: "Navbar items",
+      description: "List of items to be displayed in the navbar",
     }),
-  ),
-})
+    callToAction: Type.Optional(
+      Type.Object(
+        {
+          label: Type.String({
+            title: "Label for Call-to-Action",
+            maxLength: 30,
+          }),
+          url: Type.String({
+            title: "Call-to-Action destination",
+            description: "You can link a folder, page, or external link.",
+            format: "link",
+          }),
+        },
+        {
+          title: "Primary Call-to-Action",
+          description:
+            "You can highlight a key Call-to-Action using a prominent button.",
+        },
+      ),
+    ),
+    utility: Type.Optional(
+      Type.Object(
+        {
+          label: Type.Optional(
+            Type.String({
+              title: "Label for the list of utility links",
+              maxLength: 30,
+            }),
+          ),
+          items: Type.Array(
+            Type.Object({
+              name: Type.String({
+                title: "Name of the utility link",
+                maxLength: 30,
+              }),
+              url: Type.String({
+                title: "URL destination of the utility link",
+                format: "link",
+              }),
+            }),
+            {
+              maxItems: 4,
+            },
```

Comment:
very nit: should we set minItems 1?

Reply:
Updated in b14eb64e702debcc12cf8bf6d2383c3af8725c85

_(vaguely recall seeing JSONForms marking minItems as deprecated but should be ok, hopefully)_